webui_runner/plugs/lucet_app/shootout/wrapper.c

```diff
@@ -0,0 +1,72 @@
+#include <sightglass.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+
+#define STR1(x) #x
+#define STR(x) STR1(x)
+
+#ifdef WORKLOAD_LOCATION
+#define WASM_FILE_PREFIX STR(WORKLOAD_LOCATION)
+#else
+#error "WORKLOAD_LOCATION not defined"
+#endif
+
+
+#ifdef VM_LOCATION
+#define VM STR(VM_LOCATION)
+#else
+#error "VM_LOCATION not defined"
+#endif
+
+#define MAX_WASM_FILE_NAME_SIZE 500
+
+extern TestsConfig tests_config;
+
+TestsConfig tests_config = { .global_setup    = NULL,
+                             .global_teardown = NULL,
+                             .version         = TEST_ABI_VERSION };
+
+
+static void body_wrapper(const char *name, void *ctx)
+{
+    (void) ctx;
+    char wasm_file[MAX_WASM_FILE_NAME_SIZE];
+    snprintf(wasm_file, sizeof(wasm_file), "%s %s/%s", VM, WASM_FILE_PREFIX, name);
+    system(wasm_file);
```

Comment:
If we're running everything through a system call, do we need to duplicate the `wrapper.c` files across runtimes?